pkgs/development/python-modules/xdis/default.nix

```diff
@@ -35,20 +41,26 @@ buildPythonPackage rec {
     "xdis"
   ];
 
+  # import file mismatch:
+  # imported module 'test_disasm' has this __file__ attribute:
+  #   /build/source/pytest/test_disasm.py
+  # which is not the same as the test file we want to collect:
+  #   /build/source/test_unit/test_disasm.py
   disabledTestPaths = [
-    # Our Python release is not in the test matrix
     "test_unit/test_disasm.py"
   ];
 
   disabledTests = [
+    # AssertionError: events did not match expectation
     "test_big_linenos"
+    # AssertionError: False is not true : PYTHON VERSION 4.0 is not in magic.magics.keys
```

Comment:
These aren't great excuses for why it's ok to disable some tests, but I realize you're not the one disabling them.